magenta-lib/src/main/scala/magenta/deployment_type/GCS.scala

```diff
@@ -9,13 +9,22 @@ object GCS extends DeploymentType {
   val documentation = "For uploading files into a GCS bucket."
 
   val prefixStage =
-    Param("prefixStage", "Prefix the GCS bucket key with the target stage")
+    Param[Boolean](
```

Comment:
The change in signature for the `Param` constructor does mean we need to add an explicit type here, otherwise we get this compilation error:

```
[error] /Users/Roberto_Tyley/code/riff-raff/magenta-lib/src/main/scala/magenta/deployment_type/GCS.scala:18:10: diverging implicit expansion for type play.api.libs.json.Reads[T1]
[error] starting with method Tuple22R in trait GeneratedReads
[error]     Param(
[error]          ^
```